qp_qiime2/__init__.py

```diff
@@ -21,6 +21,20 @@
 from qiime2.sdk import PluginManager
 
 
+def _get_qiime2_type_name_and_predicate(element):
```

Comment:
Any way to have some tests for this function? I also am not sure I follow 100% what is going on. It seems like Union types have to be handled different but I don't understand what the first member is the representative member of the element.